src/redux/actions/samples/loadSamples.js

```diff
@@ -20,15 +20,15 @@ const loadSamples = (
         method: 'GET',
       });
     }
-    const json = await response.json();
+    const data = await response.json();
 
     if (!response.ok) {
-      throw new Error('HTTP status code was not 200.');
+      throw new Error(data.message);
     }
     dispatch({
       type: SAMPLES_LOADED,
       payload: {
-        samples: json[0].samples,
+        samples: data[0].samples,
```

Comment:
out of curiosity, why do we have to add the `[0]` what is in the rest of the array we receive?

Reply:
The result returns a list of samples related to a given project and experiment. `data[0]` is used because we only have one experiment per project right now. This needs to be changed if we have multiple experiments per project, each with its own combination of samples.

Reply:
I've added comments to clarify this